README.md

```diff
@@ -166,6 +166,14 @@ will create a URL like this:
 http://<podIP>:9090/metrics?foo=bar&baz=bop
 ```
 
+There are also configuration options for custom (connect and request) timeouts when querying pods for metrics:
+```yaml
+metric-config.pods.requests-per-second.json-path/request-timeout-ms: 1500
+metric-config.pods.requests-per-second.json-path/connect-timeout-ms: 500
```

Comment:
I think it would be better to call this `request-timeout` (without the `-ms`) and then use [Golang duration strings](https://golang.org/pkg/time/#Duration) for defining the value e.g. it could be defined as:

```yaml
metric-config.pods.requests-per-second.json-path/request-timeout: 15s
metric-config.pods.requests-per-second.json-path/connect-timeout: 500ms
```

Reply:
Ah yes, that is a much better way :+1: